### ISSUE TITLE
How might newsrooms coordinate document tagging with distributed contributors and maintain consistency?

### ISSUE THREAD

**Issue by eyeseast** [MuckRock/documentcloud#172]
Feedback from Lisa of CRP:
When collaborating with a large number of people, key-value data is hard to use due to users filling out keys and values inconsistently - would be useful to be able to give people a fixed list of options.

**Comment by eyeseast** [comment]
This is in the same vein as the old Assignments feature on MuckRock: https://www.muckrock.com/assignment/

More here: https://www.youtube.com/watch?v=WyeDPESUcaQ&t=2s

**Comment by eyeseast** [comment]
@DillonBergin has relevant experience doing this.

**Comment by allanlasser** [comment]
Related: #270

**Comment by DillonBergin** [comment]
Yeah, I mean I actually had an AddOn in my to-do list that would set metadata tags, key-values as a fixed list of options for a set of documents. Any reason why that would less preferable to a feature?
